examples/nextjs/styles/globals.css

```diff
@@ -16,6 +16,4 @@ div,
 p,
 span {
   color: gray;
-  border: 2px solid gray;
-  background: gray;
```

Comment:
lol this was messing with the styling on the page, even though it doesn't pierce the shadow dom, given that we wrap things with divs